erpnext/controllers/accounts_controller.py

```diff
@@ -1163,6 +1163,26 @@ def update_child_qty_rate(parent_doctype, trans_items, parent_doctype_name, chil
 	sales_doctypes = ['Sales Order', 'Sales Invoice', 'Delivery Note', 'Quotation']
 	parent = frappe.get_doc(parent_doctype, parent_doctype_name)
 
+	deleted_childs = []
+	if len(parent.items) > len(data):
```

Comment:
- remove this condition
- move deletion functionality to a new function
- deleted_children